mediation-old/src/main/java/com/wso2telco/dep/mediator/impl/ussd/USSDInboundHandler.java

```diff
@@ -52,6 +52,12 @@
  * The Class USSDInboundHandler.
  */
 public class USSDInboundHandler implements USSDHandler {
+	
+	public static final String MTCONT = "mtcont";
```

Comment:
Its better to move constant into common place, if not it will be duplicating in many palces

Reply:
done